config/prow/base/ingress.yaml

```diff
@@ -18,17 +18,24 @@ spec:
   - host: prow.calf-nursery.com
     http:
       paths:
-      - path: /
-        pathType: ImplementationSpecific
-        backend:
+      - backend:
           service:
             name: deck
             port:
               number: 80
-      - path: /hook
+        path: /*
```

Comment:
Q: is there a reason we need to point to (`/*`) for **deck** and not for other components? It used to be just `/`

Reply:
Because of the styles used by Prow, it needs to be served as well. It is also more clear what comes where this way.

Reply:
Did not fully get what you meant, but should be fine as is

/approve